docs/cmd/kn_service_create.md

```diff
@@ -39,6 +39,7 @@ kn service create NAME --image IMAGE [flags]
 ### Options
```

Comment:
Might be good to show an example?

Reply:
added an example for service create with annotation